src/assets/i18n/fr.json5

```diff
@@ -6251,6 +6310,63 @@
+"community-access-control-title": "Ce formulaire vous permet de changer les conditions d'accès de tous les Items de toutes les collections de cette communauté. Les changements peuvent être appliqués aux métadonnées des Items ou à leur contenu (Bitstreams).",
+
+//"access-control-item-header-toggle": “Item's Metadata”,
+"access-control-item-header-toggle": “Métadonnées de l'Item”,
+
+//"access-control-item-toggle.enable": "Enable option to perform changes on the item's metadata",
+"access-control-item-toggle.enable": "Activer l'option permettant d'effectuer des modifications sur les métadonnées de l'Item",
+
+//"access-control-item-toggle.disable": "Disable option to perform changes on the item's metadata",
+"access-control-item-toggle.disable": "Désactiver l'option permettant d'effectuer des modifications sur les métadonnées de l'Item",
+
+//"access-control-bitstream-header-toggle": "Bitstreams",
+"access-control-bitstream-header-toggle": "Bitstreams",
+
+//"access-control-bitstream-toggle.enable": "Enable option to perform changes on the bitstreams",
+"access-control-bitstream-toggle.enable": "Activer l'option permettant d'effectuer des modifications sur les Bitstreams",
+
+//"access-control-bitstream-toggle.disable": "Disable option to perform changes on the bitstreams",
+"access-control-bitstream-toggle.disable": "Désactiver l'option permettant d'effectuer des modifications sur les Bitstreams",
+
+//"access-control-mode": "Mode",
+"access-control-mode": "Mode",
+
+//"access-control-access-conditions": "Access conditions",
+"access-control-access-conditions": "Conditions d'accès",
+
+//"access-control-no-access-conditions-warning-message": "Currently, no access conditions are specified below. If executed, this will replace the current access conditions with the default access conditions inherited from the owning collection.",
+//"access-control-no-access-conditions-warning-message": "Aucune condition d'accès n'est spécifiée ci-dessous pour l'instant. Si exécuté, ceci remplacera les conditions d'accès actuelles par les conditions d'accès par défaut héritées de celles de la collection.",
+
+//"access-control-replace-all": "Replace access conditions",
+"access-control-replace-all": "Remplacer les conditions d'accès",
+
+//"access-control-add-to-existing":"Add to existing ones", 
+"access-control-add-to-existing":"Ajouter aux conditions d'accès existantes", 
+
+//"access-control-limit-to-specific": "Limit the changes to specific bitstreams",
+"access-control-limit-to-specific": "Limiter les modifications à des Bitstreams spécifiques",
+
+//"access-control-process-all-bitstreams": "Update all the bitstreams in the item",
+"access-control-process-all-bitstreams": "Mise à jour de tous les Bitstreams de l'item",
```

Comment:
Should be "Mise à jour de tous les Bitstreams de l'Item"

Reply:
Done in last commit. 46df7e805e2eac71c31acd35bd15367ceb5efc46